testing/infra/benchmarks.md

```diff
@@ -0,0 +1,34 @@
+# Benchmarks
+
+([https://benchmarks.ipfs.team](https://benchmarks.ipfs.team))
+
+**Status:** Running
+**Owner:** Project Operations
+
+## What is tested
+
+Times/profiles on 1-5 nodes.
+
+## How is it tested
+
+Run 1-5 nodes in a small test deployment to monitor performance changes in a small, well-controlled environment.
+
+## Inputs
+
+Benchmark scenarios.
+
+## Outputs
+
+* [x]Runtimes
+* [x]Flame Graph
+* [x]Memory/Cpu Usage
+* [ ]Pprof profiles (go)
+* [ ]Bandwidth usage
+* [ ]allocation/goroutine totals
+* [ ]Time till $event (time series)
+* [ ]Packet rate.
+
+## Targets
+
+* [x] js-ipfs
+* [ ] go-ipfs
```

Comment:
It targets go-ipfs as well but only as a way to compare (e.g. doesn't capture any DTrace probes to create Flamegraphs for go-ipfs as we don't have them in go)